src/BaselineOfNewTools/BaselineOfNewTools.class.st

```diff
@@ -231,7 +231,8 @@ BaselineOfNewTools >> packageRepositoryURLForSpec: spec [
 { #category : 'actions' }
 BaselineOfNewTools >> postload: loader package: packageSpec [
 
-	MCMethodDefinition initializersEnabled: initializersEnabled
+	MCMethodDefinition initializersEnabled: initializersEnabled.
+	Smalltalk tools fileListTool: StFileSystemPresenter.
```

Comment:
please use self class environment no direct reference to Smalltalk!

Reply:
The problem is that it is not the environment that holds the tool list so we cannot access it like this